nyx_client/nyx_client/client.py

```diff
@@ -663,6 +711,8 @@ def update_data(
         license_url: str | None = None,
         download_url: str | None = None,
         file: RawIOBase | None = None,
+        access_control: Literal["all", "none"] | None = None,
```

Comment:
same Q [as above](https://github.com/Iotic-Labs/nyx-sdk/pull/38/files#r1838273608)

Reply:
Majority of comments addressed in https://github.com/Iotic-Labs/nyx-sdk/pull/38/commits/6d8a4cbb0611334ae832680e1aaeac5c4bc41d3f